docs/index.html

```diff
@@ -8,5 +8,5 @@
     <link href="main.css" rel="stylesheet"></head>
     <body>
 	<div id="app"></div>
-    <script type="text/javascript" src="build/bundle.2753c2dd.js"></script></body>
+    <script type="text/javascript" src="build/bundle.0856d193.js"></script></body>

```

Comment:
This file isn't there.  Maybe a rebuild, or build folder not checked in?

Check the styleguide here -

https://rawgit.com/fanappics/buoy/ft-cpdt-203/docs/index.html

Reply:
Weird. I'll check that out.